1234/en.md

```diff
@@ -0,0 +1,59 @@
+# LOJ 1234 - Harmonic Number
+
+## Solution
+Instead of Brute-forcing for every individual test case, we go from lower bound (**1**) to the upper bound (**1e8**). Along the way we calculate the harmonic sum so far and any test case equal to the number we are at, assign the harmonic sum to the answer of that individual test case.
```

Comment:
I am afraid this is not descriptive enough. Let's try to elaborate a bit more and instead of focusing on how to store results (which is sort of implementation detail), let's put more focus on how to calculate the harmonic sum.

Reply:
Done!